### PR TITLE
Merge 5.5.x -> 6.0.x

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

File: acceptance/lib/puppet/acceptance/common_utils.rb [hunk 1]
```diff
@@ -1,5 +1,160 @@
...
+                                     centos, redhat: {$cron = 'cronie'}
+                                     solaris: { $cron = 'core-os' }
+                                     default: {$cron ='cron'} }
+                                     package {'cron': name=> $cron, ensure=>present, }])
+      end
+    end
+
+    module CAUtils
+
+      def initialize_ssl
```
Comment (melissa) [inline acceptance/lib/puppet/acceptance/common_utils.rb]: this method was removed from 6.0.x in https://github.com/puppetlabs/puppet/commit/6cc73d56eda68eba6e6c547c07e856f85c94e3b4#diff-48f2763d6b2f848734e74f69b76ded23

File: acceptance/lib/puppet/acceptance/common_utils.rb [hunk 2]
```diff
@@ -1,5 +1,160 @@
...
+        case host['platform']
+        when /eos/
+          on(host, "userdel #{username}", acceptable_exit_codes: [0, 1])
+        else
+          host.user_absent(username)
+        end
+      end
+    end
+
+    module CronUtils
```
Comment (melissa) [inline acceptance/lib/puppet/acceptance/common_utils.rb]: The CronUtils modules was removed from 6.0.x in https://github.com/puppetlabs/puppet/commit/0baf9ba68d3192655136f91078542109665f6fba#diff-48f2763d6b2f848734e74f69b76ded23

File: acceptance/lib/puppet/acceptance/common_utils.rb [hunk 3]
```diff
@@ -1,5 +1,160 @@
...
+        apply_manifest_on(agent, %[user { '%s': ensure => present, managehome => false }] % o[:user])
+        apply_manifest_on(agent, %[case $operatingsystem {
+                                     centos, redhat: {$cron = 'cronie'}
+                                     solaris: { $cron = 'core-os' }
+                                     default: {$cron ='cron'} }
+                                     package {'cron': name=> $cron, ensure=>present, }])
+      end
+    end
+
+    module CAUtils
```
Comment (melissa) [inline acceptance/lib/puppet/acceptance/common_utils.rb]: the rest of the CAUtils module was removed from 6.0.x in https://github.com/puppetlabs/puppet/commit/8eb7e7d09fc0ca60531a649bc662a9f12d77bd41#diff-48f2763d6b2f848734e74f69b76ded23

File: acceptance/lib/puppet/acceptance/common_utils.rb [hunk 4]
```diff
@@ -1,5 +1,160 @@
...
+
+      def user_absent(host, username)
+        case host['platform']
+        when /eos/
+          on(host, "userdel #{username}", acceptable_exit_codes: [0, 1])
+        else
+          host.user_absent(username)
+        end
+      end
+    end
```
Comment (melissa) [inline acceptance/lib/puppet/acceptance/common_utils.rb]: It looks like the only net-new stuff we should be including in for this file is Enis' addition, that ends here

Comment (McdonaldSeanp) [reply acceptance/lib/puppet/acceptance/common_utils.rb]: @melissa so everything after this line that's an addition should be removed?

**Comment by McdonaldSeanp** [comment]
@melissa does this look better?

**Pull request merged**